.github/workflows/publish-latest.yml

```diff
@@ -33,7 +33,6 @@ jobs:
                 uses: actions/setup-node@v2
                 with:
                     node-version: "16.x"
-                    cache: "yarn"
```

Comment:
lol @ykim we did have a cache?

Reply:
Apparently so. This is something GitHub provides though, so something on their end broke.